README.md

```diff
@@ -33,8 +33,9 @@ Select ***Yes*** to enable it.
 + Enter an appropriate `Title` such as ***Credit & Debit Cards***
 
 > #### API credentials
-> The following four required values, (i.e. `API Key`, `Signing Secret`, `Publishable Key - Multi Step`, and `Publishable Key - Payment Only` can be found in your ***Bolt Merchant Dashboard***  under `Settings` > `Users and Keys`
->
+> The following four required values, (i.e. `API Key`, `Signing Secret`, `Publishable Key ` can be found in your ***Bolt Merchant Dashboard***  under `Developers` > `API`
```

Comment:
nit: missing closing parenthesis

Reply:
this is now three, right? with the other publishable keys being elsewhere

Reply:
@njfaries 
nit: missing closing parenthesis
=> I've updated it

this is now three, right? with the other publishable keys being elsewhere
=> Yes